docs/src/gui/qtvcp.adoc

```diff
@@ -273,7 +273,7 @@ USER_COMMAND_FILE = CONFIGDIRECTORY/<screen_name_added_commands>
 
 If no entry is found in the _INI_, QtVCP will look in the *default path*. +
 The default path is in the configuration directory as a hidden file
-using the screen basename and rc, ie: *`CONFIGDIRECTORY/.<screen_name>rc`*
+using the screen basename and rc, ie: *`CONFIGDIRECTORY/.__<screen_name>__rc`*
```

Comment:
And here ... I stop now commenting at this point. Please build by yourself and look at the result.